extensions/amp-story/1.0/page-advancement.js

```diff
@@ -426,6 +426,21 @@ class ManualAdvancement extends AdvancementConfig {
     });
   }
 
+  /**
+   * Checks if current element is valid to trigger a tooltip.
+   * @param {!Event} event
+   */
+  isTooltipValidDescendant_(event) {
```

Comment:
How about a `canShowTooltip_` instead? With a comment saying that it has to be descendant of `amp-story-page` but not `amp-story-cta-layer`?

Reply:
Done.